### ISSUE TITLE
Editor: support toggling breakpoints on/off and removing all by menu command

### ISSUE THREAD

**Issue by ivan-mogilko** [adventuregamestudio/ags#2385]
Sometimes it may be desired to run the game without stopping at breakpoints, but keeping them for the future use.

I propose to support a "disabled" state for breakpoints, and let toggle it both individually and in batch.

1. Breakpoints may have enabled or disabled state.
2. Disabled breakpoints have a different look. For example, in MSVS they are drawn as empty circles.
3. Add menu commands:
 * Enable all breakpoints;
 * Disable all breakpoints;
 * Delete (or Remove) all breakpoints;
4. Figure out a convenient method for toggling an individual breakpoint, using a mouse, key combination or a context menu, etc.

If #2384 is implemented, I suppose that breakpoint's state also has to be serialized.

**Comment by ericoporto** [comment]
Found the first time you mentioned these. In the original idea, there was also a small GUI for these (maybe could be the same for bookmarks)

https://www.adventuregamestudio.co.uk/site/projects/open/585/
https://www.adventuregamestudio.co.uk/site/projects/open/586/
https://www.adventuregamestudio.co.uk/site/projects/open/587/

**Comment by ivan-mogilko** [comment]
Okay, I gathered all of these suggestions in this ticket.

**Comment by messengerbag** [comment]
I suppose #2454 doesn't quite fall under this issue, but I for one think it's an essential improvement for the workflow when breaking/stepping through a script.

**Comment by ericoporto** [comment]
Right, I think the list of things on top of this ticket here are just Editor stuff, add things to UI or storage format for particular things, all of which can entirely be done in C#. I would say 1, 2 and 4 are straightforward to do and 3 is hard - requires figuring out new stuff for the storage, and decisions.

This other ticket you mention is very different, since the implementation that happens behind is less clear. It looks very hard for me, simply because it's not clear how to get the behavior.
